parent/pom.xml

```diff
@@ -334,7 +334,7 @@
 								<update.site.version>${project.version} (${buildQualifier})</update.site.version>
 								<target.eclipse.version>${targetEclipseVersion}</target.eclipse.version>
 							</symbols>
-							<p2StatsUrl>http://download.jboss.org/jbosstools/usage/installs/${project.version}/${BUILD_ALIAS}/${JOB_NAME}/${buildQualifier}</p2StatsUrl>
+							<p2StatsUrl>${[2StatsUrl}</p2StatsUrl>
```

Comment:
Thanks for finding the typo. I did push another version of the commit to this PR.